silq/instrument_interfaces/keysight/E8267D_interface.py

```diff
@@ -73,6 +77,32 @@ def __init__(self, instrument_name, **kwargs):
                                      "the gate marker pulse, and end afterwards. "
                                      "This is ignored for chirp pulses where "
                                      "FM_mode = 'IQ'.")
+        self.add_parameter('I_phase_correction',
+                           unit='deg',
+                           set_cmd=None,
+                           initial_value=0,
+                           docstring="To calibrate the phase of I component independent of Q component."
+                                     "Only for FM_mode = 'IQ'.")
+        self.add_parameter('Q_phase_correction',
+                           unit='deg',
+                           set_cmd=None,
+                           initial_value=0,
+                           docstring="To calibrate the phase of Q component independent of I component."
+                                     "Only for FM_mode = 'IQ'.")
+        self.add_parameter('I_amplitude_correction',
+                           unit='V',
+                           set_cmd=None,
+                           initial_value=0,
+                           vals=vals.Numbers(min_value=-1, max_value=0),
+                           docstring="To calibrate the amplitude of I component independent of Q component."
+                                     "Takes values from -1 to 0V. Only for FM_mode = 'IQ'.")
+        self.add_parameter('Q_amplitude_correction',
+                           unit='V',
+                           set_cmd=None,
+                           initial_value=0,
+                           vals=vals.Numbers(min_value=-1, max_value=0),
+                           docstring="To calibrate the amplitude of Q component independent of I component."
+                                     "Takes values from -1 to 0V. Only for FM_mode = 'IQ'.")
```

Comment:
Added more description, have a look. -1 to 0 restriction is to make sure we don't apply above 1V to I/Q inputs.
The phase correction aims to correct any possible phase offset deviations between I and Q pulses, e.g. instead of 90 degrees we might have (instrumentally) 89 degrees phase offset, which should be corrected by these parameters and it should not depend on the frequency a lot, at least for those frequencies that you calibrate this for...